### DISCUSSION TITLE
想问下BBDown怎么安装啊

### DISCUSSION THREAD

**Discussion by Cody-Travers** [Richasy/Bili.Copilot#177] [Q&A]
点击BBDown.exe没反应，求大佬解答！

**Comment by ManyullynSword** [comment]
> 点击BBDown.exe没反应，求大佬解答！

BBDown.exe不是直接的可执行文件，你需要在Shell（包括终端、命令提示符、PowerShell，在开始菜单里搜就有）中打开，并且BBDown没有图形界面，要敲选项。至于如何加入哔哩助理，我也不知道······

**Comment by Richasy** [comment]
按照 BBDown 仓库自述文件的操作是最简单的，不需要手动下载 exe，只需要打开 PowerShell，然后运行 `dotnet tool install --global BBDown`，BBDown 就添加到你的系统运行 PATH 里了，这样你就可以直接在哔哩助理里点击下载视频。

哔哩助理会运行关联的 BBDown 命令，所以只要你能够通过命令行访问到 BBDown，那就可以在哔哩助理中下载视频

**Reply by ManyullynSword** [reply]
dotnet是有添加进环境变量，并可以正常打开，但却显示没有“tool”这个命令

**Comment by Cody-Travers** [comment]
谢谢作者大大，已经解决可以使用！！

**Reply by ManyullynSword** [reply]
为什么我无法正常执行命令

**Reply by ManyullynSword** [reply]
算了，还是手动添加到Path得了，下载测试版也好，正式版半年没更新了
